src/src/com/microsoft/aad/adal/DefaultTokenCacheStore.java

```diff
@@ -40,6 +41,8 @@
 
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
+import com.google.gson.JsonParseException;
+import com.google.gson.JsonSyntaxException;
```

Comment:
A little bit confused, i see added headers but no code to go with it. by design?